monitoring/mixin/README.md

```diff
@@ -0,0 +1,23 @@
+# Skipper Mixin
```

Comment:
as discussed, can you please check how this documentation could be better integrated with the main docs in ./docs (https://opensource.zalando.com/skipper/)

Reply:
Yeah just add it to a new section in docs/operations/operations.md there we have monitoring docs.